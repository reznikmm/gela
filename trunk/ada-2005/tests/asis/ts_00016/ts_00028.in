Associate -I../../../source/asis/spec ts_00XXX/ts_00028.ada
Action Print Any
Process_Context Compilation_Unit_Body V46f.V968
