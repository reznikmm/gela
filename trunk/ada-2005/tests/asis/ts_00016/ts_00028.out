Declaration_Kind:A_FUNCTION_BODY_DECLARATION V968 at ts_00028.ada [10:1..13:9]
  Defining_Name_Kind:A_DEFINING_IDENTIFIER V968 at ts_00028.ada [10:10..10:13]
  Definition_Kind:A_SUBTYPE_INDICATION  at ts_00028.ada [10:22..10:28]
    Expression_Kind:AN_IDENTIFIER Integer at ts_00028.ada [10:22..10:28]
  Statement_Kind:A_SIMPLE_RETURN_STATEMENT  at ts_00028.ada [12:4..12:12]
    Expression_Kind:AN_INTEGER_LITERAL 0 at ts_00028.ada [12:11..12:11]
